code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1207,7 +1207,11 @@
 			verbs |= /mob/living/carbon/human/proc/succubus_drain_finialize
 			verbs |= /mob/living/carbon/human/proc/succubus_drain_lethal
 		if(species.hard_vore_enabled == 1) //Hardvore verb.
-			verbs |= /mob/living/carbon/human/proc/shred_limb //VOREStation Edit End
+			verbs |= /mob/living/carbon/human/proc/shred_limb
+		if(species.can_fly == 1)

```

Comment:
Same here, and infact for all of the above conditionals- with binary flipping variables, `== 1` is unnecessary, redundant code.